### DISCUSSION TITLE
TypeError: generate is not a function

### DISCUSSION THREAD

**Discussion by mr-c3** [babel/babel#14815] [Q&A] [answered]
When JavaScript Babel Generate is used, the prompt is not function?

![37721e02730e98e0df192e78407027a](https://user-images.githubusercontent.com/73168492/181920013-c512a0b1-c8c0-437b-b410-c9d3a7c510ad.png)

![12d616298dc7d452afb56caf5dfe17f](https://user-images.githubusercontent.com/73168492/181920020-4a0ee3c0-469b-483e-af7e-b187ea141f93.png)

[a comment by an automated account is not shown]

**Comment by liuxingbaoyu** [comment, marked as answer]
Try `generate.default`?

**Reply by mr-c3** [reply]
23/5000 
Thank you very much. Please know whether the official document has not been updated or what

**Reply by liuxingbaoyu** [reply]
Currently babel is published as cjs, which is the case when imported in esm.
Only `parser` is a little different, there is currently an open PR, but it hasn't been updated in a long time.
https://github.com/babel/babel/pull/12565
